index.d.ts

```diff
@@ -0,0 +1,21 @@
+import Koa = require("koa");
```

Comment:
Take a look at the [Dependencies section of this page](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html).